### PR TITLE
Colony fixes

### PR DESCRIPTION
**Pull request by someaddons**
Closes #
Closes #
Closes #

# Changes proposed in this pull request:
Add zero chunk claim debug logs
Fix double colony creation
Adjust claim info command to always return the selected chunk

Review please

### CODE REVIEW

**Comment by MotionlessTrain** [comment]
Just in case, seeing that you are initialising exceptions in the logs: whenever you actually build nest to the zero chunk, it is hopefully not going to crash?

File: gradle.properties [hunk 1]
```diff
@@ -3,7 +3,7 @@ org.gradle.daemon=false
 
 modId=minecolonies
 modGroup=com.ldtteam
-modVersion=0.0.1-LOCAL
+modVersion=1.1.87-BETA
 modBaseName=minecolonies
```
Comment (Raycoms) [inline gradle.properties]: Cough

File: src/main/java/com/minecolonies/coremod/commands/colonycommands/CommandShowClaim.java [hunk 2]
```diff
@@ -36,10 +36,10 @@ public int onExecute(final CommandContext<CommandSourceStack> context)
         final ServerLevel level = context.getSource().getLevel();
 
         // Colony
-        BlockPos pos = context.getSource().getEntity().blockPosition();
+        BlockPos pos = BlockPos.containing(context.getSource().getPosition());
```
Comment (Raycoms) [inline src/main/java/com/minecolonies/coremod/commands/colonycommands/CommandShowClaim.java:39]: can just get blockPosition from an entity too

**Review by Raycoms — DISMISSED** [review]

**Comment by Raycoms** [comment]
> Just in case, seeing that you are initialising exceptions in the logs: whenever you actually build nest to the zero chunk, it is hopefully not going to crash?

No, its fine,t hats just to get the trace

File: src/main/java/com/minecolonies/coremod/commands/colonycommands/CommandShowClaim.java (hunk 2 above)
Comment (someaddons) [reply src/main/java/com/minecolonies/coremod/commands/colonycommands/CommandShowClaim.java:39]: the source is better as it allows the command to run from commandline^^

**Pull request merged**